From 885226a405ab80405b55f7dfca869028290ff61a Mon Sep 17 00:00:00 2001
From: Emil Medve <Emilian.Medve@Freescale.com>
Date: Fri, 17 Jan 2014 10:35:39 -0600
Subject: [PATCH 082/466] fsl_qman: Remove unneeded semicolon

This fixes the following coccinelle warning:

Unneeded semicolon

Signed-off-by: Emil Medve <Emilian.Medve@Freescale.com>
Change-Id: I87a5d2bec27c59e7ae7c6bf381e2f0c718f95dc3
Reviewed-on: http://git.am.freescale.net:8181/8111
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Haiying Wang <Haiying.Wang@freescale.com>
Reviewed-by: Emilian Medve <Emilian.Medve@freescale.com>
[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]
Signed-off-by: Bin Jiang <bin.jiang@windriver.com>
---
 drivers/staging/fsl_qbman/bman_low.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/fsl_qbman/bman_low.h b/drivers/staging/fsl_qbman/bman_low.h
index 32871fa..9872001 100644
--- a/drivers/staging/fsl_qbman/bman_low.h
+++ b/drivers/staging/fsl_qbman/bman_low.h
@@ -519,6 +519,6 @@ static inline int bm_shutdown_pool(struct bm_portal *p, u32 bpid)
 			stop = true;
 		} else
 			++aq_count;
-	};
+	}
 	return 0;
 }
-- 
1.7.10.4

